client/my-sites/themes/themes-magic-search-card/index.jsx

```diff
@@ -230,7 +230,7 @@ class ThemesMagicSearchCard extends React.Component {
 	}
 
 	render() {
-		const { isJetpack, translate, filters } = this.props;
```

Comment:
We can also drop the `isJetpackSite` selector from the `connect()` further down in this file.